LibraryTests/QuantumPhaseEstimationTests.qs

```diff
@@ -0,0 +1,54 @@
+namespace Microsoft.Quantum.Tests {
+
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+	open Microsoft.Quantum.Extensions.Convert;
+    open Microsoft.Quantum.Extensions.Testing;
+	open Microsoft.Quantum.Extensions.Math;
+
+
+    /// # Summary
+    /// Assert that the QuantumPhaseEstimation operation for the T gate
+	/// return 0000 in the controlRegister when targetState is 0 and
+	/// return 0010 when the targetState is 1  
+    operation QuantumPhaseEstimation_Test () : ()
+    {
+        body
+        {
+            let Oracle = DiscreteOracle(T_PhaseEstimation(_,_));
```

Comment:
Our style guide suggests that variable names generally should follow the camelCase convention (no initial capitalization, no underscores).

Reply:
The `(_, _)` notation here for partial application is correct, but isn't actually needed; the bare operation name `T_PhaseEstimation` is a value whose type is `(Int, Qubit[]) => () : Adjoint, Controlled`. It might be good to remove the `(_, _)` notation here, since no partial application is actually being performed in this case.